kommit.rb

```diff
@@ -21,10 +21,14 @@ def initial_info
   @chance_to_commit_on_sunday = ask_for('What should be the chance of commiting on Sundays? (percentage)').to_i
 end
 
+def git_repo?()
+  File.directory?('.git')
```

Comment:
As I wanted this project to create a repository in a predictable place, I've decided that it would be in the home directory of the user. This line would work but it would check in the current directory and not the home directory of the user so it needs some changes in order to work as expected

Reply:
File.directory?("#{Dir.home}/kommitr_commits") should be enough to do the trick